frontend/src/concepts/analyticsTracking/segmentIOUtils.tsx

```diff
@@ -108,6 +110,10 @@ export const fireIdentifyEvent = (properties: IdentifyEventProperties): void =>
       isAdmin: properties.isAdmin,
       canCreateProjects: properties.canCreateProjects,
     };
-    window.analytics.identify(properties.anonymousID, traits);
+    if (properties.userID) {
+      window.analytics.identify(properties.userID, traits);
+    } else {
+      window.analytics.identify(properties.anonymousID, traits);
+    }
```

Comment:
```suggestion
    window.analytics.identify(properties.userID || properties.anonymousID, traits);
```

Although this works, shouldn't the `anonymousID` be set to `userID` in the first place instead? That way we don't bother generating the `anonymousID` and there's always only a single way to deal with the ID.

Doesn't sandbox also want the `toolchain.dev.openshift.com/sso-account-id` as part of the identify payload?

Reply:
Wrt the latter, we only need the web-user-id, which is the sso-id. Actually for my account the 2 seem to even be the same